packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -217,6 +223,15 @@ export default class Import extends Component {
     )
   }
 
+  _importVmFromUrl = () => {
+    const url = this.state.url
+    const file = {
+      name: url.split('/').pop(),
+    }
+    const type = file.name.split('.').pop().trim()
```

Comment:
Should we add a way to force the type in case the URL path isn't properly formatted but the user knows what type it is?